nx/source/services/capsa.c

```diff
@@ -238,58 +238,82 @@ static Result _capsaReadImageDataFromAlbumMovieReadStream(u64 stream, s64 offset
 }
 
 Result capsaOpenAlbumMovieStream(u64 *stream, const CapsAlbumFileId *file_id) {
+    if (hosversionBefore(4,0,0))
+        return MAKERESULT(Module_Libnx, LibnxError_IncompatSysVer);
+
     Result rc=0;
 
-    if (!serviceIsActive(&g_capsaAccessor)) rc =_capsaOpenAccessorSessionForApplication(&g_capsaAccessor);
+    if (!serviceIsActive(&g_capsaAccessor)) rc =_capsaOpenAccessorSession(&g_capsaAccessor);
 
     if (R_SUCCEEDED(rc)) rc = _capsaOpenAlbumMovieReadStream(stream, file_id);
 
     return rc;
 }
 
 Result capsaCloseAlbumMovieStream(u64 stream) {
+    if (hosversionBefore(4,0,0))
```

Comment:
Mention sysver in the .h for these then?

Reply:
Isn't the one above capsaOpenAlbumMovieStream enough until real descriptions follow?

Reply:
sure